src/product.html

```diff
@@ -0,0 +1,271 @@
+                    sadipscing
+                    elitr, sed diam nonumy.</p>
+            </div>
+            <div class="service__item">
+                <img src="../img/3.png" class="service__icon" />
+                <p class="service__subtitle">Security and privacy</p>
+                <p class="service__text">Lorem ipsum dolor sit amet, consetetur
+                    sadipscing
+                    elitr, sed diam nonumy.</p>
+            </div>
+            <div class="service__item">
+                <img src="../img/3.png" class="service__icon" />
+                <p class="service__subtitle">Security and privacy</p>
+                <p class="service__text">Lorem ipsum dolor sit amet, consetetur
+                    sadipscing
+                    elitr, sed diam nonumy.</p>
+            </div>
+            <div class="service__item">
+                <img src="../img/3.png" class="service__icon" />
+                <p class="service__subtitle">Security and privacy</p>
+                <p class="service__text">Lorem ipsum dolor sit amet, consetetur
+                    sadipscing
+                    elitr, sed diam nonumy.</p>
+            </div>
+            <div class="service__item">
+                <img src="../img/3.png" class="service__icon" />
+                <p class="service__subtitle">Security and privacy</p>
+                <p class="service__text">Lorem ipsum dolor sit amet, consetetur
+                    sadipscing
+                    elitr, sed diam nonumy.</p>
+            </div>
+            <div class="service__item">
+                <img src="../img/3.png" class="service__icon" />
+                <p class="service__subtitle">Security and privacy</p>
+                <p class="service__text">Lorem ipsum dolor sit amet, consetetur
+                    sadipscing
+                    elitr, sed diam nonumy.</p>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="about__item">
+            <div class="about__background">
+            </div>
+            <p
+                class="about__title about__title--small">
+                Why should you work with us?</p>
+            <p class="about__title">To upscale your business to the
+                next
+                level</p>
+            <p class="about__text">Lorem ipsum dolor sit amet, consetetur sadipscing elitr,
+                sed diam
+                nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut
+                labore.</p>
+        </div>
+        <div class="about__item">
+            <img src="../img/bg1.png" class="about__img" />
+        </div>
+    </section>
+    <section class="about">
+        <div class="about__item">
+            <img src="../img/bg1.png" class="about__img" />
+        </div>
+        <div class="about__item">
+            <div class="hero-background">
```

Comment:
This is not BEM. A block cannot have another block inside. `hero-background` shouldn't be in the `about` block.